js/views/components/ModeratorsStatus.js

```diff
@@ -0,0 +1,55 @@
+import BaseVw from '../baseVw';
+import loadTemplate from '../../utils/loadTemplate';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    const opts = {
+      className: 'moderatorStatus',
+      ...options,
+      initialState: {
+        hidden: true,
+        showSpinner: true,
+        spinnerTimer: true,
+        showLoadBtn: false,
+        loaded: 0,
+        total: 0,
+        mode: 'loaded',
+        ...(options.initialState || {}),
+      },
+    };
+
+    super(opts);
+    this.options = opts;
+  }
+
+  events() {
+    return {
+      'click .js-browseMore': 'clickBrowseMore',
+    };
+  }
+
+  clickBrowseMore() {
+    this.trigger('browseMore');
+  }
+
+  remove() {
+    clearTimeout(this.loadingTimer);
+    super.remove();
+  }
+
+  render() {
+    loadTemplate('components/moderatorsStatus.html', (t) => {
+      this.$el.html(t({
+        ...this.getState(),
+      }));
+      if (this._state.showSpinner && this._state.spinnerTimer) {
```

Comment:
On any state change render is called, so with this code you'd be potentially setting multiple timeouts which turn off the spinner. I think you'd want to clear the timeout at the start of render() to avoid that.